index.html

```diff
@@ -231,6 +231,22 @@
 
 
     // HISTOGRAMS
```

Comment:
`NaN` means "not a number." It's a hint that you tried to use something like it was a number, but it wasn't.

What do you think is in `d.x0`? Try `console.log()`-ing it.

Also note that you're setting `histogram.bins()` twice. The second one will override the first one (which is good, because the second one is correct).

Also note that the only reason to use groups (`g`) here would be if you plan on adding labels and such. If you only ever have a the bars themselves, you could just append a `rect` directly.